docs/using/installing/index.rst

```diff
@@ -4,17 +4,27 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The following diagram illustrates the architecture of the cluster you are about to install:
```

Comment:
In my opinion not in this case, and particularly not when it directly contradicts the diagram which it describes - that shows Flocker Cluster separate to the CLI.